### ISSUE TITLE
ModifierWithoutDefault and overriden functions

### ISSUE THREAD

**Issue by manuel-martos** [twitter/compose-rules#86]
I've seen that composable functions with `Modifier` param in interface doesn't need default value, but any code that extends that interface is flagged by this rule. For instance: 

```kotlin
interface Bleh {
    @Composable
    fun Something(modifier: Modifier)
}

class BlehImpl : Bleh {
    @Composable
    override fun Something(modifier: Modifier) {}
}
```

is flagged and the only solution for it is to add an exception in detekt baseline file. Shouldn't be nice to add and exception for this scenario?

**Comment by mrmans0n** [comment]
Seems like an oversight.
Do you mind submitting a PR for https://github.com/twitter/compose-rules/blob/main/rules/common/src/main/kotlin/com/twitter/compose/rules/ComposeModifierWithoutDefault.kt#L17 where you add the exemption? 

```diff
-        if (function.definedInInterface || function.isActual) return
+        if (function.definedInInterface || function.isActual || function.isOverride) return
```

Extra marks for adding the test to detekt/ktlint `ComposeModifierWithoutDefaultCheckTest` files XD

**Comment by nuhkoca** [comment]
Same for abstract functions
